setup.sh

```diff
@@ -12,6 +12,35 @@ sudo apt install -y python3-pip
 echo "installing flask"
 pip3 install flask
 
+echo "installing flask restful"
+pip3 install flask-restful
+
+echo "installing flask jwt"
+pip3 install flask-jwt-extended
+
+echo "installing passlib"
+pip3 install passlib
+
+echo "installing flask-sqlalchemy"
+pip3 install flask-sqlalchemy
+
+echo "installing psycopg2-binary"
+pip3 install psycopg2-binary
+
```

Comment:
Shouldn't all of these pip3 installs should be in the requirements file.

Reply:
ack